openrtb_ext/request.go

```diff
@@ -168,7 +169,14 @@ type ExtIncludeBrandCategory struct {
 	TranslateCategories *bool  `json:"translatecategories,omitempty"`
 }
 
+type MediaTypePriceGranularity struct {
+	Banner *PriceGranularity `json:"banner,omitempty"`
+	Video  *PriceGranularity `json:"video,omitempty"`
+	Native *PriceGranularity `json:"native,omitempty"`
+}
+
 // PriceGranularity defines the allowed values for bidrequest.ext.prebid.targeting.pricegranularity
+// or bidrequest.ext.prebid.targeting.mediatypepricegranularity
```

Comment:
Should this comment live above the definition of `MediaTypePriceGranularity` on line 172?

e.g: `MediaTypePriceGranulairty defines allowed values for bidrequest.ext.prebid.targeting.mediatypepricegranularity`

Reply:
Added comment for `MediaTypePriceGranularity`. 

Modified this comment to this:
```go
// PriceGranularity defines the allowed values for bidrequest.ext.prebid.targeting.pricegranularity
// or bidrequest.ext.prebid.targeting.mediatypepricegranularity.banner|video|native
```
Idea here is to show that same structure is used for both `pricegranularity` and `mediatypepricegranularity.banner|video|native`